src/pykeen/nn/emb.py

```diff
@@ -192,6 +192,35 @@ class Embedding(RepresentationModule):
 
     This class provides the same interface as :class:`torch.nn.Embedding` and
     can be used throughout PyKEEN as a more fully featured drop-in replacement.
+
+    It extends it by adding additional options for normalizing, constraining, or applying dropout.
+
+    When a *normalizer* is selected, it is applied in every forward pass. It can be used, e.g., to ensure that the
+    embedding vectors are of unit length. A *constrainer* can be used similarly, but it is applied after each parameter
+    update (using the post_parameter_update hook), i.e., outside of the automatic gradient computation.
+
+    The optional dropout can be used as a regularization technique. Moreover, it enables to obtain uncertainty estimates
```

Comment:
what about the regularizer kwarg itself?

Reply:
Dropout is a different technique, cf. the description here https://pytorch.org/docs/stable/generated/torch.nn.Dropout.html

The existing regularizers in `Embedding` add an additional regularization term to the loss function, and hence prevent through the gradient of the regularization term that weights become too large. Dropout, in constrast, randomly sets some entries to zero, thus preventing the model to focus too strongly on an individual component (the description is somewhat short for a complex topic :sweat_smile: ).

Reply:
Thanks, maybe you could include something like that in the docstring as well.

Reply:
6000e1d ?